### PR TITLE
Adds Research Outpost (100% Done)

### PR DESCRIPTION
**Pull request by Sarcalogo**
Refer to this. 

https://github.com/HippieStationCode/HippieStation13/pull/2127

### CODE REVIEW

**Comment by Nebyab** [comment]
okay

**Comment by Sarcalogo** [comment]
AHHHHHHHHH just found a really dumb bug. lame, it was working before... not sure what happened.

File: code/game/area/Space Station 13 areas.dm [hunk 1]
```diff
@@ -1414,55 +1482,6 @@ var/list/teleportlocs = list()
 	icon_state = "storage"
 
 
-// Gamma Vessel
```
Comment (TDcoolguy300) [inline code/game/area/Space Station 13 areas.dm:1417]: Why are you removing the Gamma areas?

**Comment by trifornt-tinfornt** [comment]
Because the Gamma Research Derelict is pure trash? I get that it might not be what this PR is supposed to do but having 3 .357 revolvers up for easy grabs on something that you can literally teleport to on roundstart is just terrible design.

**Comment by TDcoolguy300** [comment]
Fuck off Trifornt, without those areas the template will fuck up the whole system

**Comment by trifornt-tinfornt** [comment]
At least make the area actually work then, because getting numerous .357 revolvers at roundstart is no.

**Comment by TDcoolguy300** [comment]
Leave that to another discussion Trifornt, as far as i am concerned this is a DNM until that area is readded

**Comment by trifornt-tinfornt** [comment]
A'ight.
Readd if it it'll break the game then.

**Comment by Sarcalogo** [comment]
Uh... it's not that I removed the Gamma area's, it's that I started to work on this BEFORE the Gamma areas were even added, meaning, there's no way I would have had them in.

**Comment by TDcoolguy300** [comment]
Still needs to be readded before the DNM can be removed

**Comment by TDcoolguy300** [comment]
code/game/area/Space Station 13 areas.dm:1800:error: Vessel: expected end of statement

code/game/area/Space Station 13 areas.dm:1839:error: expected expression

**Comment by Sarcalogo** [comment]
That should do it.

**Comment by TDcoolguy300** [comment]
I'm satisfied now, DNM removed.

**Comment by Lawligagger** [comment]
cool

**Comment by Sarcalogo** [comment]
Also here are some pictures of the completed station.. probably should have included that earlier.

http://imgur.com/a/DI3Kk

**Comment by TDcoolguy300** [comment]
Appears to be fine, Neb and Lawli approved.

**Pull request merged**